docs/spark-procedures.md

```diff
@@ -587,3 +587,75 @@ Get all the snapshot ancestors by a particular snapshot
+| `table`       | ✔️ | string | Name of the table to create changlog view                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `changelog_view`        |   | string | Name of the view to create                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `options`     |   | map<string, string> | A map of Spark read options to use. For example, `start-snapshot-id`, the snapshot id to start reading from exclusively. If not provided, the table’s first snapshot will be used as the starting point. `end-snapshot-id`, the snapshot id to stop reading at inclusively. Default to the current snapshot. `start-timestamp`, the timestamp to start reading from exclusively. If not provided, the table’s first snapshot will be used as the starting point.`end-timestamp`, the timestamp to stop reading at inclusively. If not provided, the table’s current snapshot will be used as the ending point. | 
+|`compute_updates`| | boolean | Whether to compute updates. Defaults to false                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | 
```

Comment:
This is a bit confusing. I think looking at this from an outside perspective i'm not sure what "compute_updates" would mean. Ideally all of our descriptions should be a more descriptive than a restatement of the name of the column.